src/test/java/org/folio/marc4ld/service/marc2ld/preprocessor/impl/DataField043PreprocessorTest.java

```diff
@@ -46,7 +46,7 @@ void preprocessShouldPreprocessSubfieldA() {
     var expectedDataField = createDataField(List.of(createSubfield('a', "n-us"), createSubfield('k', "a---")));
     when(factory.newDataField("043", ' ', ' ')).thenReturn(createDataField(List.of()));
     when(factory.newSubfield('a', "n-us")).thenReturn(createSubfield('a', "n-us"));
-    when(dictionaryProcessor.getValue("NAME", "n-us")).thenReturn(Optional.of("United States"));
+    when(dictionaryProcessor.getValue("GEOGRAPHIC_CODE_TO_NAME", "n-us")).thenReturn(Optional.of("United States"));
```

Comment:
for some reason these tests pass if you use ```mvn clean install```

Reply:
Very strange.